ClassAttendance/src/ConsoleUI/Views/Interfaces/IMainMenuView.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ConsoleUI.Views.Interfaces
+{
+    public interface IMainMenuView
```

Comment:
Why we do not have a single `IMenuView` interface?

Reply:
Fixed